src/desktop/components/commercial_filter/filters/period/period_map.coffee

```diff
@@ -15,7 +15,6 @@ module.exports =
     'Late 19th Century',
     'Mid 19th Century',
     'Early 19th Century',
-    '18th Century & Earlier',
```

Comment:
PR creep: we want to remove this filter from the collect page as well.

Reply:
We recently removed this from the new collect page in favor of hardcoding the values -- not sure what other places this time period filter is used but maybe we should all be using the same list? https://github.com/artsy/reaction/blob/master/src/Apps/Collect/Components/ArtworkGrid/index.tsx#L315

Reply:
Cool good to know!